packages/SketchMorph2-Core.package/M2Backend.class/instance/currentGeometricForm..st

```diff
@@ -1,3 +1,3 @@
-accessing
-currentGeometricForm: aSymbol
-	currentGeometricForm := aSymbol
\ No newline at end of file
+delegation
+currentGeometricForm: anObject
+	self context currentGeometricForm: anObject
```

Comment:
We decided not to write the variable type in methods in the delegation category since these methods are only supposed to pass the method call to the context without having knowledge of the type of data they are receiving.